File: src/main/java/vip/yazilim/libs/springvip/rest/model/RestErrorResponse.java
package vip.yazilim.libs.springvip.rest.model;

/**
 * @author Emre Sen, 24.07.2019
 * @contact maemresen@yazilim.vip
 */
public class RestErrorResponse extends RestResponse<RestError<?>> {

    public RestErrorResponse() {
        super(true);
    }
}

File: src/main/kotlin/vip/yazilim/libs/springvip/util/generic/rest/model/RestErrorResponse.kt
package vip.yazilim.libs.springvip.util.generic.rest.model

/**
 * The model that RestController used as a response when an error occurred
 *
 * @author Emre Sen - maemresen@yazilim.vip
 * 24.07.2019
 */
class RestErrorResponse(timestamp: Long
                        , path: String
                        , message: String
                        , data: RestError<*>)
    : RestResponse<RestError<*>>(true, timestamp, path, message, data);
